.github/workflows/test-js.yaml

```diff
@@ -1,4 +1,5 @@
-name: '[Javascript] tests'
+name: '[js] Tests'
```

Comment:
Given that the FTML badge title is `[<project>] language`, I'd prefer something like `]frontend] JS` or `[web] JS` for consistency. The URL for the badge on the top README will also need to be updated